src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/host-form/host-form.component.ts

```diff
@@ -94,7 +93,7 @@ export class HostFormComponent extends CdForm implements OnInit {
         validators: [CdValidators.ip()]
       }),
       labels: new UntypedFormControl([]),
-      maintenance: new UntypedFormControl(false)
+      maintenance: new UntypedFormControl()
```

Comment:
why this change?

Reply:
Hi @nizamial09 , thanks for brining in. While I was making the changes in the Ui . I found that if we checked the maintenance checkbox, the value we are getting of status: 'maintenance' otherwise its blank and here we are getting "500 internal server error" as status should be boolean one. 


Can you please confirm if status should be **boolean or string**?

Reply:
it should be a boolean.

Reply:
now thatt i checked the code properly, the value of `this.status` is a string. but the form control value is boolean since its a checkbox. the conversion happen here: https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/host-form/host-form.component.ts#L148